### PR TITLE
Fix #1410 --  display none class removed for print scss

### LINKED ISSUE #1410: BLOCKER - The page’s second column content is hidden in print stylesheets.
> Blocker: Critical issue. Certain users cannot perceive or operate this, with no workaround available.

This makes it impossible for users of magnifiers to access this content ([22:00](https://www.youtube.com/watch?v=dAiMPTOMh1U&t=1320s)).

### PR DESCRIPTION
**Pull request by Bhagyarsh**

### CODE REVIEW

**Comment by CuriousLearner** [comment]
Hi @Bhagyarsh 

Can you please also post a screenshot of the site after your changes?

**Comment by Bhagyarsh** [comment]
Before: Not able to see  email fields and the right sidebar  
image
[before.pdf](https://github.com/django/djangoproject.com/files/13062759/before.pdf)

After: Able to see email fields and the right sidebar  
image
[after.pdf](https://github.com/django/djangoproject.com/files/13062765/after.pdf)

File: djangoproject/scss/_print.scss [hunk 1]
```diff
@@ -89,5 +89,9 @@
     .list-news li {
         margin-top: 0;
     }
-
+#homepage,
+    [role="complementary"],
+    .form-input {
+        display: block !important;
```
Comment (sabderemane) [inline djangoproject/scss/_print.scss:95]: Is it really necessary to add `!important` ?

Comment (Bhagyarsh) [reply djangoproject/scss/_print.scss:95]: https://github.com/django/djangoproject.com/pull/1420/files/1ef11dae922b4927c42fcc44f129201614704496#diff-1ae87a3c7f843d4ccaa6330d04803e4679855dd15bc5276526d071081b78c112L70-R72

Comment (Bhagyarsh) [reply djangoproject/scss/_print.scss:95]: Yes, as we have  `display: none !important;` on line number 71

**Pull request merged**